_posts/2020-04-24-news2.md

```diff
@@ -0,0 +1,10 @@
+---
+layout: post
+title: "Rút ngắn thời gian làm việc của bưu điện - COVID-19 đã lan rộng sang 6 tỉnh thành mới"
+category: news
+tags: 
+  - toàn quốc
+---
+Nhằm ngăn chặn sự lây lan của COVID-19, bưu điện đã cho rút ngắn thời gian làm việc của gần 1 vạn cơ sở thuộc 13 tỉnh thành nằm trong danh sách "các tỉnh thành yêu cầu cảnh giác đặc biệt". Mặt khác, mặc dù cây rút tiền tự động của ngân hàng bưu điện (Yucho) vẫn hoạt động như thường lệ nhưng ngân hàng bưu điện cũng khuyến cáo người dân nên đeo khẩu trang và điều chỉnh giờ tới cây rút tiền để tránh bị lây nhiễm.
+
+Nguồn: [Báo NHK](https://www3.nhk.or.jp/news/html/20200424/k10012402571000.html)
```

Comment:
Bài này dịch chất lượng không ổn lắm. Đại ý của bài theo anh hiểu:

- Chính sách rút ngắn thời gian làm việc đã có từ ngày 22 ở 7 tỉnh công bố tình trạng khẩn cấp đặc biệt
- Do có thêm 6 tỉnh công bố tình trạng khẩn cấp đặc biệt, nên bưu điện cũng mở rộng việc áp dụng chính sách rút ngắn thời gian tại 6 tỉnh trên.

Đoạn lược dịch hiện tại đọc có cảm giác như đây là chính sách mới hoàn toàn. Với cả thông tin chính xác giờ làm việc cũng nên được nêu ra. Tiêu đề cần được thay đổi.

Reply:
đã sửa